ui/components/multichain/edit-networks-modal/edit-networks-modal.js

```diff
@@ -0,0 +1,83 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { TextVariant } from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getNonTestNetworks, getTestNetworks } from '../../../selectors/index';
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  Checkbox,
+  Text,
+  Box,
+} from '../../component-library';
+import { NetworkListItem } from '../index';
+
+export const EditNetworksModal = ({ onClose }) => {
+  const t = useI18nContext();
+  const nonTestNetworks = useSelector(getNonTestNetworks);
+  const testNetworks = useSelector(getTestNetworks);
+  return (
+    <Modal
+      isOpen
+      onClose={() => {
+        onClose();
+      }}
+      className="import-nfts-modal"
```

Comment:
this className refers to import-nfts, it probably should be edit-networks?

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/26097/commits/59624c3ec6a12be6e6f22970bbe3ace3aa54574a